_data/header-footer-content.yml

```diff
@@ -4,8 +4,8 @@ header:
   ctas:
   -
     link: "/about/"
-    h2: "Measurement Lab is a partnership between New America's Open Technology Institute, Google Inc., Princeton University's PlanetLab, and other supporting partners."
-    h3: "Learn more about M-Lab"
+    h2: "M-Lab is led by teams based at Code for Science & Society; New America’s Open Technology Institute; Google, Inc; Princeton University’s PlanetLab; and supported by partners around the world."
```

Comment:
> M-Lab is

This should probably say 'Measurement Lab is...'

Reply:
ok. will change and update the PR with an additional commit